public/docs/ts/latest/guide/displaying-data.jade

```diff
@@ -179,65 +187,70 @@ figure.image-display
 
-  At the moment, we're binding to an array of strings. We do that occasionally in real applications, but
-  most of the time we're displaying objects &mdash; potentially instances of classes.
+  At the moment, we're binding to !{_an} !{_array} of strings. We do that occasionally in real applications, but
+  most of the time we're binding to more specialized objects.
 
-  Let's turn our array of hero names into an array of `Hero` objects. For that we'll need a `Hero` class.
+  Let's turn our !{_array} of hero names into !{_an} !{_array} of `Hero` objects. For that we'll need a `Hero` class.
 
-  Create a new file in the `app/` folder called `hero.ts` with the following short bit of code.
-+makeExample('displaying-data/ts/app/hero.ts', null, 'app/hero.ts')(format = ".")
+  Create a new file in the `!{_appDir}` folder called  <ngio-ex path="hero.ts"></ngio-ex> with the following code:
+  
++makeExcerpt('app/hero.ts')
 
-:marked
-  We've defined a class with a constructor and two properties: `id` and `name`.
+block hero-class
+  :marked
+    We've defined a class with a constructor and two properties: `id` and `name`.
 
-  It might not look like we have properties, but we do. We're taking
-  advantage of a TypeScript shortcut in our declaration of the constructor parameters.
+    It might not look like we have properties, but we do. We're taking
+    advantage of a TypeScript shortcut in our declaration of the constructor parameters.
 
-  Consider the first parameter:
-+makeExample('displaying-data/ts/app/hero.ts', 'id-parameter')
+    Consider the first parameter:
 
-:marked
-  That brief syntax does a lot:
-  * declares a constructor parameter and its type
-  * declares a public property of the same name
-  * initializes that property with the corresponding argument when we "new" an instance of the class
+  +makeExcerpt('app/hero.ts ()', 'id')
+
+  :marked
+    That brief syntax does a lot:
+    * Declares a constructor parameter and its type
+    * Declares a public property of the same name
+    * Initializes that property with the corresponding argument when we "new" an instance of the class
 
 .l-main-section
 :marked
   ## Using the Hero class
-  Let's redefine the `heroes` property in our component to return an array of these Hero objects
-  and also set the `myHero` property with the first of these mock heroes.
-+makeExample('displaying-data/ts/app/app.component.3.ts', 'heroes', 'app.component.ts (excerpt)')(format=".")
+  Let's make the `heroes` property in our component return !{_an} !{_array} of these `Hero` objects.
+
++makeExcerpt('app/app.component.3.ts', 'heroes')
 
 :marked
   We'll have to update the template.
-  At the moment it displays the entire `hero` object, which used to be a string value.
-  Let's fix that so we interpolate the `hero.name` property.
-+makeExample('displaying-data/ts/app/app.component.3.ts', 'template','app.component.ts (template)')(format=".")
+  At the moment it displays the `Hero` object.
```

Comment:
`Hero` object doesn't seem quite right here. Revert back to "the string value of the `Hero` object"?